app/frontend/styles/components/step-by-step-navigation.scss

```diff
@@ -0,0 +1,642 @@
+    @include govuk-media-query($from: tablet) {
+      margin-bottom: 60px;
+    }
+  }
+}
+
+.app-step-nav__controls {
+  padding: 3px 3px 0 0;
+  text-align: right;
+}
+
+.app-step-nav__button {
+  color: $govuk-link-colour;
+  cursor: pointer;
+  background: none;
+  border: 0;
+  margin: 0;
+
+  &:focus {
+    @include govuk-focused-text;
+  }
+}
+
+.app-step-nav__header:hover .app-step-nav__button:not(:focus) {
+  color: $govuk-link-colour;
+}
+
+// removes extra dotted outline from buttons in Firefox
+// on focus (standard yellow outline unaffected)
+.app-step-nav__button::-moz-focus-inner {
+  border: 0;
+}
+
+.app-step-nav__button--title {
+  @include govuk-font(19, $weight: bold);
+  display: inline-block;
+  padding: 0;
+  text-align: left;
+  color: govuk-colour("black");
+
+  .app-step-nav--large & {
+    @include govuk-font(24, $weight: bold);
+  }
+}
+
+.app-step-nav__button--controls {
+  @include govuk-font(14);
+  position: relative;
+  z-index: 1; // this and relative position stops focus outline underlap with border of accordion
+  padding: 0.5em 0;
+  text-decoration: underline;
+
+  &:hover:not(:focus) {
+    color: $govuk-link-hover-colour;
+  }
+
+  .app-step-nav--large & {
+    @include govuk-font(16);
+  }
+}
+
+.app-step-nav__steps {
+  padding: 0;
+  margin: 0;
+}
+
+.app-step-nav__step {
+  position: relative;
+  padding-left: 45px;
+  list-style: none;
+}
+
+.app-step-nav__step:after {
+  content: "";
+  position: absolute;
+  z-index: 2;
+  width: 0;
+  height: 100%;
+  border-left: solid 2px $govuk-border-colour;
+  background: #fff;
+  left: 0;
+  margin-left: 12px;
+  top: 15px;
+}
+
+@include govuk-media-query($from: tablet) {
+  .app-step-nav--large .app-step-nav__step {
+    padding-left: 60px;
+  }
+
+  .app-step-nav--large .app-step-nav__step:after {
+    left: 0;
+    margin-left: 16px;
+    border-width: 3px;
+    top: 30px;
+  }
+}
+
+.app-step-nav__step:last-child:before {
+  content: "";
+  position: absolute;
+  z-index: 6;
+  bottom: 0;
+  left: 0;
+  margin-left: 6.5px;
+  width: 13px;
+  height: 0;
+  border-bottom: solid 2px $govuk-border-colour;
+}
+
+.app-step-nav__step:last-child:after {
+  height: -webkit-calc(100% - 15px);
+  height: calc(100% - 15px);
+}
+
+.app-step-nav__step:last-child .app-step-nav__help:after {
+  height: 100%;
+}
+
+@include govuk-media-query($from: tablet) {
+  .app-step-nav--large .app-step-nav__step:last-child:before {
+    margin-left: 8.75px;
+    width: 17.5px;
+    border-width: 3px;
+  }
+
+  .app-step-nav--large .app-step-nav__step:last-child:after {
+    height: calc(100% - 30px);
+  }
+}
+
+.app-step-nav__step--active .app-step-nav__circle--number,
+.app-step-nav__step--active .app-step-nav__help:after,
+.app-step-nav__step--active .app-step-nav__substep:after,
+.app-step-nav__step--active:after,
+.app-step-nav__step--active:last-child:before {
+  border-color: govuk-colour("black");
+}
+
+.app-step-nav__circle {
+  -webkit-box-sizing: border-box;
```

Comment:
should not be necessary to add any vendor prefixes as postcss has autoprefixer `/Users/alexbowen/Projects/teacher-vacancy-service/postcss.config.js`. whether this configured correctly is another matter. so leave these prefixes in here for the time being. i am compiling a todo list for css and js, so i will add this to it and this can addressed at a future date